lib/puppet/provider/hbm.rb

```diff
@@ -125,15 +122,15 @@ def res_create
       execute(c)
     end
 
-    unless resource[:provider].to_s == 'config'
+    unless resource[:provider].to_s == 'config' # rubocop:disable Style/GuardClause # FIXME: unsure what rubocop prefers
```

Comment:
Looking at https://stackoverflow.com/questions/31153934/guard-clause-instead-of-wrapping-the-code-inside-a-conditional-expression-rails/64492257#64492257 it seems like it would like it to be something like this:
```
return if resource[:provider].to_s == 'config'

members = getmembers(resource[:provider], resource[:name], resource[:members])
...
```

Maybe?

Reply:
I can try this, but I would feel more comfortable if there would be tests. Unsure if I am able to add them. But I will try this.

I would anyway recommend to test these changes with some real life clients to see if everything still works as expected.

Reply:
Sounds sane to me!

Reply:
After changing the code to your suggestion this error occurs:
```
pdk (ERROR): rubocop: unexpected token kEND
(Using Ruby 2.4 parser; configure using `TargetRubyVersion` parameter, under `AllCops`) (lib/puppet/provider/hbm.rb:152:1)
```

Reply:
Still unsure what rubocop wants from me. Will leave it as it is for now.